Extensions/Settings/Dnn.PersonaBar.SiteSettings/admin/personaBar/App_LocalResources/SiteSettings.resx

```diff
@@ -1072,7 +1072,7 @@
     <value>Enable Localized Content</value>
   </data>
   <data name="EnableLocalizedContentHelpText.Text" xml:space="preserve">
-    <value>Enabling localized content allows you to provide translated module content in addition to displaying translated static text. Once localized contetnt is enabled the default site culture will be permanently set and <strong>cannot be changed</strong>.</value>
+    <value />
```

Comment:
It looks like this content was removed inadvertently?

Reply:
Yes. This should be reverted.